pkg/glance/volumes.go

```diff
@@ -73,35 +78,22 @@ func getInitVolumeMounts() []corev1.VolumeMount {
 			MountPath: "/var/lib/config-data/default",
 			ReadOnly:  true,
 		},
-		{
-			Name:      "config-data-custom",
-			MountPath: "/var/lib/config-data/custom",
```

Comment:
shouldn't we keep this, or do you merge also the custom configs into main service conf? can start glance with --config-dir=DIR as the default is none/

Reply:
So I was just following the pattern we're now using the new Keystone and Placement operators, where the custom config is actually provided in-line within the CR and then added to the config data CM like so: https://github.com/abays/glance-operator/blob/af3f68396b87827483fe660600fb6f74516731eb/controllers/glanceapi_controller.go#L504-L542.  I guess I need to mount a volume pointing to the `custom.conf` key in the config data CM, whereas in the old Glance operator there was an actual separate "custom config data" CM.